api/routes/api.php

```diff
@@ -18,3 +18,17 @@
     Route::post('/tickets', 'createTicket');
 });
 
+Route::get('/test-notify', function () {
+    try {
+        Notify::sendEmail(
+            'email@domain.com', // Replace with the email you gave to @esizer
+            'c386053b-e65c-477e-8f09-da19c2e9dea0',
```

Comment:
I know this is just an experimental branch, but the `templateId` would probably be best as an environment variable as well, oui?

Reply:
So, the way it works, we would need to add a lot of template id's and since they aren't really secret and need to scale I don't think we need to be putting them in as environment variables. What we could do is maybe make a map like we did `ApiEnums` so it is more readable. i.e

```php
<?php

namespace App\Facades;

class NotifyEnums
{
    const STATUS_UPDATE_TEMPLATE = 'template-id';

    public static function templates(): array
    {
        return [
            self::STATUS_UPDATE_TEMPLATE
        ];
    }
}

# Usage
$templates = NotifyEnums::templates();
Notify::sendEmail(
    'email@domain.com',
    NotifyEnums::STATUS_UPDATE_TEMPLATE,
    []
);
```

Reply:
I like that, much easier to read than an id.